client/shipping-label/state/test/actions.js

```diff
@@ -0,0 +1,76 @@
+import { expect } from 'chai';
+import { openPrintingFlow } from '../actions';
+import sinon from 'sinon';
+
+const context = {
+	storeOptions: {
+		countriesData: {
+			US: '',
+		},
+	},
+};
+
+const getState = function() {
+	return {
+		shippingLabel: {
+			form: {
+				origin: {
+					values: {
+						address: 'Some street',
+						postcode: '',
+						state: 'CA',
+						country: 'US',
+						phone: '123',
+					},
+					ignoreValidation: false,
+					selectNormalized: false,
+					isNormalized: false,
+					normalized: '',
+				},
+				destination: {
+					values: {
+						address: 'Some street',
+						postcode: '',
+						state: 'CA',
+						country: 'US',
+						phone: '123',
+					},
+					ignoreValidation: false,
+					selectNormalized: false,
+					isNormalized: false,
+					normalized: '',
+				},
+				packages: {
+					selected: {},
+				},
+				rates: {
+					values: {},
+				},
+			},
+		},
+	};
+};
+
+describe( 'Shipping label Actions', () => {
+	describe( '#openPrintingFlow', () => {
+		describe( 'when ignoring validation for addresses', () => {
+			let spy = sinon.spy();
```

Comment:
Sure - renamed to `dispatchSpy`.

> Also, if this were called with an unexpected action (not the 3 below) - what happens?

The tests will still pass if it's called with other actions. I also added some safeguards to make sure it's not called with any of those those steps using a different (untested) code path.